modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/site_settings/details.jsp

```diff
@@ -105,89 +105,37 @@ if (parentGroupId != GroupConstants.DEFAULT_PARENT_GROUP_ID) {
 	<aui:input disabled="<%= disabled %>" helpMessage='<%= disabled ? "this-site-cannot-inherit-the-content-from-its-parent-site-since-the-parent-site-is-already-inheriting-the-content-from-its-parent" : StringPool.BLANK %>' inlineLabel="right" labelCssClass="simple-toggle-switch" name="inheritContent" type="toggle-switch" value="<%= value %>" />
 </c:if>
 
-<h4 class="text-default"><liferay-ui:message key="membership-options" /></h4>
-
 <c:if test="<%= !siteGroup.isCompany() %>">
-	<aui:select label="membership-type" name="type">
-		<aui:option label="open" value="<%= GroupConstants.TYPE_SITE_OPEN %>" />
-		<aui:option label="restricted" value="<%= GroupConstants.TYPE_SITE_RESTRICTED %>" />
-		<aui:option label="private" value="<%= GroupConstants.TYPE_SITE_PRIVATE %>" />
-	</aui:select>
 
 	<%
-	boolean manualMembership = true;
+	String title = StringPool.BLANK;
 
-	if (liveGroup != null) {
-		manualMembership = GetterUtil.getBoolean(liveGroup.isManualMembership(), true);
+	if (parentGroup != null) {
+		title = HtmlUtil.escape(parentGroup.getDescriptiveName(locale)) + " (" + LanguageUtil.get(request, parentGroup.getTypeLabel()) + ")";
 	}
 	%>
 
-	<aui:input inlineLabel="right" label="allow-manual-membership-management" labelCssClass="simple-toggle-switch" name="manualMembership" type="toggle-switch" value="<%= manualMembership %>" />
+	<label class="control-label"><liferay-ui:message key="parent-site" /></label>
 
-	<%
-	List<Group> parentGroups = new ArrayList<Group>();
+	<div class="input-group mb-3">
+		<div class="input-group-item">
+			<input class="field form-control lfr-input-text" id="<portlet:namespace />parentSiteTitle" name="<portlet:namespace />parentSiteTitle" readonly="<%= true %>" value="<%= title %>" />
 
-	if (parentGroup != null) {
-		parentGroups.add(parentGroup);
-	}
-	%>
+			<aui:input name="parentGroupSearchContainerPrimaryKeys" type="hidden" value="<%= parentGroupId %>" />
```

Comment:
This line adds a form input. Please check:

* The input is validated on the processing action

* The backend is protected agains value forging



Refer to documentation: